tests/Behat/WebtoolsGlobanContext.php

```diff
@@ -46,19 +47,19 @@ public function backupGlobanConfigs(): void {
   }
 
   /**
-   * Make sure that load.js has the correct query parameters.
+   * Asserts the Webtools Globan JSON snippet on the page.
    *
-   * @param string $globan_option
-   *   The globan option value.
-   * @param string|null $globan_lang
-   *   The globan language option value.
+   * @param string $value
+   *   The expected JSON string.
    *
-   * @Then the Webtools javascript is loaded with the globan options :globan_option
-   * @Then the Webtools javascript is loaded with the globan options :globan_option and language :globan_lang
+   * @throws \Exception
+   *
+   * @Then the page should have globan json snippet :value
```

Comment:
I think we can finally improve this, fetch the script that contains `"utility":"globan"`, read its values and compare them with the ones provided in a table:
```
Then the Webtools globan should be loaded with the values: 
| name | value |
| theme | dark |
| logo | false |
|...|
```

What do you think?

Reply:
I had this idea already but currently, I already noticed that behat always pass from behat step data as string including from table. JSON and parsed data more strict according to data type. For sure we can massage data from behat or JSON to some unified data type (convert all values to string (if JSON data) or cast values from behat Table.
But it looks a bit more tricky and anyway more accurate to compare full json string.

Reply:
True, it would be more hacky. Let's keep it this way.